src/dba/AbstractModelFactory.class.php

```diff
@@ -855,7 +855,7 @@ public function getDB($test = false) {
     }
     else {
       global $CONN;
-      $dsn = 'mysql:dbname=' . $CONN['db'] . ";host=" . $CONN['server'] . ";port=" . $CONN['port'] . ";charset=utf8";
+      $dsn = 'mysql:dbname=' . $CONN['db'] . ";host=" . $CONN['server'] . ";port=" . $CONN['port'] . ";charset=utf8mb4";
```

Comment:
Add a comment with the following comment:
```
# The utf8mb4 is here to force php to connect with utf8, so you can save emoji's or other non ascii chars into the database. If you are running into issues with this line, we could make this configurable.
```

Reply:
Also add an item to the changelog.md, you might need to create a header here for the new version.

Reply:
Done.